### DISCUSSION TITLE
Install libraries to use offline

### DISCUSSION THREAD

**Discussion by Davidvx10** [thebaselab/codeapp#763] [Q&A] [answered]
I know the app can run code offline, but when I want to import libraries in a script it will obvilously not work. Is there a way that I can save or install some libraries in my ipad so that the 'import' doesn't look for the libraries online?

**Comment by bummoblizard** [comment, marked as answer]
Once you installed the libraries, they work offline. Which language you are working on?

**Reply by Davidvx10** [reply]
python, I'm a begginer though. Do you have a link to a video or article that says how to install some libraries?

**Reply by bummoblizard** [reply]
https://code.thebaselab.com/guides/your-first-program-in-python

You can also refer to tutorials for desktop Python. Code App uses the standard pip command to manage libraries.

**Reply by Davidvx10** [reply]
Thanks, I really appreciate it.
